packages/esm-patient-chart-app/src/visit/visit-form/visit-form.component.tsx

```diff
@@ -497,8 +497,10 @@ const StartVisitForm: React.FC<StartVisitFormProps> = ({
     return [maxVisitStartDatetime, minVisitStopDatetime];
   }, [visitToEdit]);
 
-  const visitStartDate = getValues('visitStartDate');
-  minVisitStopDatetime = minVisitStopDatetime ?? Date.parse(visitStartDate.toLocaleString());
+  const visitStartDateValue = getValues('visitStartDate');
+  const visitStartDate = visitStartDateValue !== undefined ? visitStartDateValue : new Date();
+  const minVisitStopDatetimeFallback = Date.parse(visitStartDate.toLocaleString());
+  minVisitStopDatetime = minVisitStopDatetime || minVisitStopDatetimeFallback;
```

Comment:
```suggestion
  const visitStartDate = getValues('visitStartDate') ?? new Date();
  minVisitStopDatetime = minVisitStopDatetime ?? Date.parse(visitStartDate.toLocaleString());
```

Isn't this the whole change you did?

Reply:
hI @vasharma05 ,Here is an updated screen shot 
![Screenshot from 2024-01-29 14-25-18](https://github.com/openmrs/openmrs-esm-patient-chart/assets/48015736/ac899754-99c3-49a4-b0e6-35d1ae92f3c0)

Reply:
Hi @denniskigen @vasharma05 . i addressed your comments. let me know your suggestions.